src/api/helpers/formatEntityData.ts

```diff
@@ -145,7 +222,8 @@ export function getEditionBasicInfo(edition: any) {
 			height: _.get(edition, 'height', null),
 			languages: getLanguages(edition),
 			pages: _.get(edition, 'pages', null),
-			releaseEventDates: _.get(edition, 'releaseEventSet.releaseEvents', []).map((event) => event.date),
+			publisherSet: filterPublisherSet(edition),
```

Comment:
Generally I try to avoid presenting technical terminology like 'set' to end users, for whom it will make less sense:
```suggestion
			publishers: filterPublisherSet(edition),
```

In our case, that means we want to return publisherSet.publishers directly, as I don't think we return the set id (which is an internal id) in any of the other set types.
Look for example at how we return the aliasSet.aliases in `getEntityAliases` for comparison